CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## February
+- New processors
+  - GetFileProperties
+- Release 1.0.4
```

Comment:
Is this release actually going to get cut? We don't have a `1.0.3` yet.

Reply:
My reply from Slack: `
```
Dave Borncamp  12:49 PM
Was the 1.0.4 release that's mentioned in the changelog actually cut? I don't see a 1.0.3

Seth Lasky  1:17 PM
the version’s been changed in pom, I don’t know if we are automatically pushing the nar on version change or if we need to be doing it manually

Dave Borncamp  1:18 PM
I think Lucas was doing that manually

Seth Lasky  1:20 PM
I wouldn’t consider the actual push to be the release though, the changelog is for the project at large so if anyone wants to build a nar using the current version of the project it will be 1.0.4
```